### ISSUE TITLE
Modus Navbar-Accessibility The search tooltip is not Accessible using the tab navigation

### ISSUE THREAD

**Issue by hsoni177** [trimble-oss/modus-web-components#1968]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-web-components/blob/main/CONTRIBUTING.md)

### Describe the issue

To replicate this issue enable the `show-search` and set the `searchTooltip={text:'Search'};` prop for the search button. Navigate to the search button using the tab key. On the first tab, the tooltip is not displayed you have to press the tab key one more time to see the tooltip on the search button.

### What is the issue regarding ?

@trimble-oss/modus-web-components

### Priority

Medium

### What product/project are you using Modus Components for ?

e-Builder

### What is your team/division name ?

e-Builder

### Are you willing to contribute ?

Yes
